Updates.list

```diff
@@ -20,8 +21,16 @@
 3075851 5 Windows Update Client for Windows 7 and Windows Server 2008 R2: August 2015
 3075853 5 Windows Update Client for Windows 8.1 and Windows Server 2012 R2: August 2015
 3080149 9 Update for "customer experience" and "diagnostic telemetry".
+3081954 9 This update adds telemetry data points to Work Folders for Asimov telemetry pipeline in Windows 7
+3083324 9 Windows 10 preparation.
+3083710 9 Windows 10 preparation.
+3102810 9 Fixes an issue regarding long wait while searching for Windows Updates but also has Windows 10 upgrade preparation for Windows 7.

```

Comment:
Between 7-8 hours wait for update search to finish and an off chance that it adds something related to Win10 upgrades, I'll take my chances.